dashboard/README.md

```diff
@@ -1,20 +1,63 @@
 # Dashboard
 
-This folder contains all code and resources required for the dashboard.
+This folder contains all code and resources required to extract the 
+files from the s3 bucket and load it into the dashboard with visualisations.
 
-## Set-up and installation instructions
+## Configure environment
 
-1. Create venv `python3 -m venv venv`
-2. Activate venv `source .\venv\bin\activate`
-3. Install Requirements `pip install -r requirements.txt`
-4. Requires environment variables:
+```sh
+python3 -m venv venv
+source ./venv/bin/activate
+pip3 install -r requirements.txt
+```
 
-- DATABASE_NAME
-- DATABASE_USERNAME
-- DATABASE_PASSWORD
-- DATABASE_IP
-- DATABASE_PORT
+## Configure environment variables
 
-## Development Instructions
+The following environment variables must be supplied in a `.env` file.
 
+`BUCKET_NAME`
+`ACCESS_KEY_ID`
+`SECRET_ACCESS_KEY`
+`DATABASE NAME`
+`DATABASE USERNAME`
+`DATABASE PASSWORD`
+`DATABASE_IP`
+`DATABASE_PORT`
+
+## Run the code
+To run the entire pipeline
+- Run `streamlit run pipeline.py`
+
+To run just the extract_s3.py
+- Run `python3 extract_s3.py`
+
+To run just the streamlit_app.py
 - Run `streamlit run streamlit_app.py`
+
+# Docker image
+
+Build the docker image
+
+```sh
+docker build -t plant-dashboard . --platform "linux/amd64"
+```
+
+Run the docker image locally
+
+```sh
+docker run -it --env-file .env -p 8501:8501 plant-dashboard 
+```
+
+## Files
+
+### extract_s3.py
+
+Contains all code and resources required for extracting the plant data from an s3 bucket.
+To run the file individually : `python3 extract_s3.py`
+
+### streamlit_app.py
+
```

Comment:
could add a comment about why we have displayed 10 plants each. Axis names -- why we chose to fix at certain numbers. Maybe explain why we thought a toggle button was useful (user friendly, more control etc)